.github/workflows/cd.yaml

```diff
@@ -1,7 +1,8 @@
 name: Release
 
 on:
-  push:
+  pull_request:
+    types: [closed]
```

Comment:
Will this run on a PR that is closed but not merged? e.g. if someone raises a malicious PR and we close it

Reply:
line 12 checks that is was merged